examples/opensora_pku/README.md

```diff
@@ -313,7 +313,7 @@ Then please revise `scripts/text_condition/train_videoae_221x512x512.sh`, and ch
 
 ```bash
 # start 221x512x512 finetuning, 8 NPUs
-bash scripts/text_condition/train_videoae_221x512x512.sh
```

Comment:
Since 221x512x512 optimizer parallelism leads to oom, how about deleting `train_videoae_221x512x512.sh` and only keep `train_videoae_221x512x512_sp.sh` ?

Reply:
The main branch doesn't seem to provide script `train_videoae_221x512x512.sh` ?

Reply:
Yes, you are right.